MatrixKit/Models/Account/MXKAccountManager.m

```diff
@@ -58,6 +60,9 @@ - (instancetype)init
     {
         _storeClass = [MXFileStore class];
         
+        // Migrate old account file to new format
+        [self migrateAccounts];
```

Comment:
In case of migration, the account will loaded twice but this is not an issue.

Reply:
I've tried to use `NSKeyedUnarchiver::decodeObject:` instead of `NSKeyedUnarchiver::decodeObjectForKey` so I could just read the data of the old file and then encrypt and save the result to the new file. `NSKeyedUnarchiver::decodeObject:` returns `nil` in this case unfortunately.